.github/workflows/docker-build-publish.yml

```diff
@@ -14,13 +14,20 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+-arabica"
       - "v[0-9]+.[0-9]+.[0-9]+-mocha"
   pull_request:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "The checkout reference (ie tag, branch, sha)"
+        required: true
+        type: string
 
 jobs:
   docker-security-build:
     permissions:
       contents: write
       packages: write
-    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.4.3 # yamllint disable-line rule:line-length
+    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@checkout_ref # yamllint disable-line rule:line-length
```

Comment:
NOTE: added this specific branch reference since the PR with the change is still open. Once merged the branch will be deleted so this would break. This should be updated to the new version of the .github repo once https://github.com/celestiaorg/.github/pull/112 is merged.